jenkins/test-history/gen_html.py

```diff
@@ -91,11 +126,15 @@ def gen_tests(data, prefix, exact_match):
             else:
                 status = 'okay'
             test_html.append('<li class="suite">')
+            if most_recent_failure:
+                test_html.append(gubernator_href(suite, most_recent_failure, test))
```

Comment:
Here it'd be nicer to append `"<a href=\"%s\">" % gubernator_href(...)` so that it clearly matches the `</a>` below.